common/djangoapps/student/models.py

```diff
@@ -753,6 +753,15 @@ def user_post_save_callback(sender, **kwargs):
             ceas = CourseEnrollmentAllowed.for_user(user).filter(auto_enroll=True)
 
             for cea in ceas:
+                # skip enrolling already enrolled users
```

Comment:
lets add a detailed description of the bug and link here

Reply:
Discussed over video, let's keep the comment here simple - whoever is looking into why this block of code was implemented will probably git blame it, which should lead to the PR with more details of the bug.